internal/git.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package internal
+
+import "net/url"
+
+// Exported variables whose purpose is to get injected at build-time using:
+//
+// go build -ldflags "-X gopkg.in/DataDog/dd-trace-go.v1/internal.gitRepositoryURL=$(git config --get remote.origin.url)
```

Comment:
Wouldn't this be weird from a user perspective? Why aren't we setting this by means of regular config like everything else? That means: env. vars, tracer config, etc. In a normal deployment on the user application this would end up containing paths like `-X vendor/gopkg.in/DataDog/dd-trace-go.v1/internal.gitRepositoryURL=$(git config --get remote.origin.url)` etc. Seems a bit weird to address variables in vendored libraries...

Reply:
I'd rather have that discussion in the related RFC : )

https://docs.google.com/document/d/1Jy_A3qUC0uvz5J4iP3OiON72CbfkW7T0K2c4ODdRP1A/edit#heading=h.14th4i9fjqox